src/styles/blocks/snake.scss

```diff
@@ -0,0 +1,40 @@
+.snake {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+
+  @media (min-width: 900px) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  &__child {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: #fff;
+    height: 300px;
+
+    &--1 {
+      background-color: rgb(100%, 0%, 0%);
+    }
+
+    &--2 {
+      background-color: rgb(80%, 0%, 0%);
+    }
+
+    &--3 {
+      background-color: rgb(60%, 0%, 0%);
+    }
+
+    &--4 {
+      background-color: rgb(40%, 0%, 0%);
+    }
+
+    &--5 {
+      background-color: rgb(20%, 0%, 0%);
+    }
+
+    &--6 {
+      background-color: rgb(0%, 0%, 0%);
+    }
```

Comment:
Let's find another solution for this logic. Don't create additional classes. Here you have almost the same styles for every class.

Hint: use `nth-child: ` + `@for` cylce